src/bin/server.ts

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env node
-
-import { exit } from "process";
 import { resolve } from "path";
 import { Driver } from "zwave-js";
 import { ZwavejsServer } from "../lib/server";
 import { createMockDriver } from "../mock";
 import { parseArgs } from "../util/parse-args";
 
+const normalizeKey = (
+  key: string,
+  keyName: string,
+  supportOZWFormat: boolean = false
+): Buffer => {
+  if (key.length === 32) return Buffer.from(key, "hex");
```

Comment:
```suggestion
  if (Buffer.isBuffer(key)) return key; 
  if (key.length === 32) return Buffer.from(key, "hex");
```

Optionally check the length, but that's [already done](https://github.com/zwave-js/node-zwave-js/blob/a92dfd8761e7d8a535779276fffb242384b73a31/packages/zwave-js/src/lib/driver/Driver.ts#L252) in the driver.